### ISSUE TITLE
Enable WordPress authentication on science.mozilla.org

### ISSUE THREAD

**Issue by alanmoo** [mozilla/science.mozilla.org#559]
This will allow us to access draft posts so that we can enable preview support in #554 . (Draft posts require being logged in as an authenticated user to access. This will likely also require some server side logic to handle Oauth.

This seems like it will be a separate authentication layer from the planned Github auth, I'm not sure what else we'd be able to use this for other than previews. @acabunoc maybe you have some insight?

(I'm not sure this is something worth pursuing as I investigate deeper, but wanted to open the ticket in order to start a conversation)

**Comment by acabunoc** [comment]
I don't think it would have any use other than previews since we only give wordpress logins to staff/fellows.

I workaround this by changing the preview url -- but it previews on WordPress is always formatted differently when we actually post.

**Comment by alanmoo** [comment]
Oh interesting. Perhaps I could put the science site stylesheets in the WordPress preview and make the preview button go there...
